canonical-web-and-design/maas-ui
# feat(ui) machine details commissioning details view

## Done
- Point commissioning details to a commissioning details view, reusing the test details components
- Made the back from the details view dynamic based on the section you are in
- Updated the path to the React version of commissioning
- ~~**Drive by** Add `is-dense` to TableMenu to stop it jumping so much while loading~~ ***Reverted and [issue filed](https://github.com/canonical-web-and-design/maas-ui/issues/2172)***
- **Drive-by** Right align the back to testing view link on test details view
- **Drive-by** Manage more icon states in the commissioning table
-  **Drive-by** Fixes links to testing from tests

## QA

### MAAS deployment
To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch
You can run this branch by:

- Run with `dotrun`

### QA steps
- Go to a machine details
- Switch to commissioning
- Find and click a test that has "View details"
- See that the details match bolla
- Click back link and check it returns you to the commissioning view
- Do the same for tests and check the return link works

## Fixes
Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/2213
